pages/credits.vue

```diff
@@ -0,0 +1,266 @@
+
+      <div class="content is-size-4">
+        <p>
+          Northern Climate Reports would not be possible without contributions
+          from a wide variety of climate science professionals. These include
+          ecosystem modelers and boreal ecologists who interpret and look for
+          patterns in massive amounts of data, plus geospatial data analysts and
+          software developers who verify the data and create visualizations that
+          aid understanding. Science communicators and administrative staff
+          round out the team, using graphic design, writing, and organizational
+          skills to help this tool work for a wide variety of users.
+        </p>
+      </div>
+    </div>
+
+    <div class="person mt-5 columns">
+      <div class="column is-3 is-offset-2">
+        <img src="~assets/images/people/breen_bw.jpg" />
+      </div>
+      <div class="column is-5">
+        <h3 class="subtitle is-3">Amy Breen</h3>
+        <div class="content is-size-5">
+          <p>
+            Dr. Amy Breen is an expert in Arctic vegetation ecology, short- and
+            long-term effects of climate change and disturbance on vegetation.
+            Her research interests include field studies and modeling to
+            understand and forecast changes in arctic terrestrial ecosystems,
+            particularly the response of vegetation to disturbance and climate
+            change.
+          </p>
+          <p>
+            <a href="https://www.researchgate.net/profile/Amy-Breen"
+              >Learn more about Dr. Breen</a
+            >
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="person mt-5 columns">
+      <div class="column is-3 is-offset-2">
+        <img src="~assets/images/people/littell3_bw.jpg" />
+      </div>
+      <div class="column is-5">
+        <h3 class="subtitle is-3">Jeremy Littell</h3>
+        <div class="content is-size-5">
+          <p>
+            Dr. Jeremy Littell is a Research Ecologist with the Alaska Climate
+            Adaptation Science Center (CASC). He conducts research on the role
+            of climate and ecological drought in Alaska and other forested
+            ecosystems. He also facilitates the use of climate information in
+            planning, adaptation, and vulnerability assessment.
+          </p>
+          <p>
+            Littell&rsquo;s background is the ecoclimatology of western North
+            America's forests. His doctoral work at the University of Washington
+            focused on ecological and climatic controls on wildfire in the
+            western US and the role of climate in Douglas-fir tree growth across
+            its climatic range. He also worked as a research scientist at the
+            University of Washington Climate Impacts Group, collaborating with
+            resource managers in Federal and state agencies to better understand
```

Comment:
```suggestion
            resource managers in federal and state agencies to better understand
```
"federal" is not capitalized in Nancy's bio. I suggest making it lowercase in Jeremy's bio also.